packages/models-library/src/models_library/aiodocker_api.py

```diff
@@ -55,6 +55,6 @@ class AioDockerTaskSpec(TaskSpec):
 class AioDockerServiceSpec(ServiceSpec):
     TaskTemplate: AioDockerTaskSpec | None = None
 
-    class Config(ServiceSpec.Config):
```

Comment:
Careful with this change.
NOTE that the `ServiceSpec.Config` does not get automatically inherited and it needs to be done this way.

Reply:
You are right, returned back with `# type: ignore`. In Pydantic v2 there will be better ways how to approach this.